VS_Project/Pokerapp/Pokerapp/Web.config

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Waarom staat jouw Web.config hierin? Is je .gitignore wel up-to-date? (Zo niet, kopieer hem even van master)